views/FanView/FanView.tsx

```diff
@@ -74,31 +86,69 @@ export const FanView: FC<FanViewProps> = ({ route }) => {
   return (
     <View style={styles.container}>
       <Navbar text={`${location} - ${nodeId} - ${fan?.name}`} />
-      <View style={styles.content}>
-        <Typography
-          variant="body-medium"
-          text={`Aktualny temperatura wentylacji: ${fan.values.V_TEMP}`}
-        />
-        <Input text={temp} onChange={onTempChange} />
-        <Typography
-          variant="body-medium"
-          text={`Aktualny prędkość wentylacji: ${fan.values.V_DIRECTION}`}
-        />
-        <Input text={direction} onChange={onDirectionChange} />
-        <Typography
-          variant="body-medium"
-          text={`Aktualny kierunek głowicy wentylacji: ${fan.values.V_PERCENTAGE}`}
-        />
-        <Input text={percentage} onChange={onPercentageChange} />
-        <Button
-          text="Zmień parametry wentylatora"
-          onPress={handleChangeFanParams}
-        />
-        <Button
-          text="Pobierz najnowsze wartości"
-          onPress={handleGetFanParams}
-        />
-      </View>
+      <ScrollView>
+        <View style={styles.content}>
+          <View style={styles.section}>
+            <Typography
+              variant="body-medium"
+              text={`Aktualna temperatura wentylacji: ${fan.values.V_TEMP}°C`}
+            />
+            <Typography
+              variant="body-small"
+              text={`Zakres: ${fanRangeValues.minTemp} - ${fanRangeValues.maxTemp}`}
+            />
+            <Input
+              text={temp}
+              onChange={onTempChange}
+              placeholder=""
+              keyboardType="numeric"
+              max={fanRangeValues.maxTemp}
+            />
+          </View>
+          <View style={styles.section}>
+            <Typography
+              variant="body-medium"
+              text={`Aktualna prędkość wentylacji: ${fan.values.V_PERCENTAGE}%`}
+            />
+            <Typography
+              variant="body-small"
+              text={`Zakres: ${fanRangeValues.minPercentage} - ${fanRangeValues.maxPercentage}`}
+            />
+            <Input
+              text={percentage}
+              onChange={onPercentageChange}
+              placeholder=""
+              keyboardType="numeric"
+              max={fanRangeValues.maxPercentage}
+            />
+          </View>
+          <View style={styles.section}>
+            <Typography
+              variant="body-medium"
+              text={`Aktualny kierunek głowicy: ${fan.values.V_DIRECTION}°`}
+            />
+            <Typography
+              variant="body-small"
+              text={`Zakres: ${fanRangeValues.minDirection} - ${fanRangeValues.maxDirection}`}
+            />
+            <Input
+              text={direction}
+              onChange={onDirectionChange}
+              placeholder=""
```

Comment:
One more, why did you add an empty placeholder? Just removed it in all places :D

Reply:
Will change placeholder prop to empty initially